src/app/pages/component-sidenav/component-nav.html

```diff
@@ -0,0 +1,20 @@
+<div class="docs-component-viewer-nav">
+  <div class="docs-component-viewer-nav-content">
+    <nav *ngFor="let category of docItems.getCategories((params | async)?.section); let last = last;">
+      <button (click)="toggleExpand(category.id)">
```

Comment:
* Add an `aria-label` like "Expand the {{category.name}} section of navigation links"
* Add `aria-controls` pointing to the element that expands
* Add `aria-expanded`